src/global/components/messages-queue.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <TransitionGroup
+    class="messages--container"
+    name="messages-queue"
+    tag="div"
+  >
+    <FeedbackMessage
+      v-for="message in globalStore.messages"
+      :key="message.content + message.type + message.title"
```

Comment:
Zminimalizowałeś tutaj ryzyko, ale one nadal istnieje. Mogą być wiadomosci z tym samym typem, tytułem i contentem. Na przyszłośc warto pamiętać że jeżeli klucze będą złe, nieunikalne lub będą zmienne (np. index w loopie jest bardzo złym pomysłem) to mogą wystąpić problemy z niepotrzebnym nadmiarowym rerenderowaniem listy